hw/ip/kmac/data/kmac.hjson

```diff
@@ -625,17 +625,39 @@
       regwen: "CFG_REGWEN"
       fields: [
         { bits: "HashCntW-1:0"
-          name: "threshold"
-          desc: "Hash Threshold"
-        }
-        { bits: "HashCntW+15:16"
           name: "hash_cnt"
           desc: "Hash (KMAC) counter"
           swaccess: "ro"
           hwaccess: "hwo"
```

Comment:
Since we are taking out the threshold field here, can we move these hwaccess/swaccess attributes up to the main register? There is no need anymore to declare two differing attributes at the reg and field level.

Reply:
good point. I'll have a look.

Reply:
see latest force-push